web/html/src/branding/css/susemanager/components/cveaudit.less

```diff
@@ -0,0 +1,10 @@
+.scan-data-indicator {
```

Comment:
This isn't really a global component in the same sense that e.g. buttons or inputs are, it would be preferable to make this an isolated module instead. See for example `web/html/src/manager/content-management/shared/components/panels/filters-project/filters-project.module.css` and how it's used in code. Note that you will need to use your style in the component code differently once you make this change.